.local/bin/workman

```diff
@@ -58,6 +58,13 @@ if [ ! -f ./${WORKMAN_CONFIG_FILE} ]; then
     echo "BUILD_COMMAND=\"python2 x.py build -j8\""
     echo "# Command to run after creating a new working directory."
     echo "AFTER_COMMAND=\"rustup toolchain link \$(basename \$(pwd))-stage1 ./build/x86_64-unknown-linux-gnu/stage1\""
+    echo
+    echo "# Command to run in order to continue working in a given working directory"
+    echo "WORK_ATTACH_COMMAND=\"tmux new -A -s \\\"\\\${PROJECT_NAME}/\\\${TASK}\\\"\""
+    echo "# Command to run in order to start working in a given working directory"
+    echo "WORK_NEW_COMMAND=\"tmux new -s \\\"\\\${PROJECT_NAME}/\\\${TASK}\\\"\""
```

Comment:
We can re-use the `SESSION_REATTACH_COMMAND` here for `tmux` since `-A` will have no-effect when it's a new session and it doesn't already exist. Without `-A`,  it would previously have exited with an error had the session already existed, but I don't mind if it doesn't do that.
```suggestion
    echo "SESSION_ATTACH_COMMAND=\"${SESSION_REATTACH_COMMAND}\"\""
```